app/src/main/java/com/CMPUT301F21T30/Habiteer/Session.java

```diff
@@ -2,45 +2,40 @@
 
 import static android.content.ContentValues.TAG;
 
-import static androidx.core.content.ContextCompat.startActivity;
-import static java.lang.Boolean.FALSE;
-import static java.lang.Boolean.TRUE;
-
 import com.CMPUT301F21T30.Habiteer.ui.habit.Habit;
 
-import android.app.Activity;
-import android.app.Application;
 import android.content.Context;
 import android.content.Intent;
 import android.util.Log;
 import android.widget.Toast;
 
 import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
 
-import com.CMPUT301F21T30.Habiteer.ui.habit.ListHabitViewModel;
+import com.CMPUT301F21T30.Habiteer.ui.habitEvents.Event;
 import com.google.android.gms.tasks.OnFailureListener;
 import com.google.android.gms.tasks.OnSuccessListener;
-import com.google.firebase.firestore.CollectionReference;
 import com.google.firebase.firestore.DocumentReference;
 import com.google.firebase.firestore.DocumentSnapshot;
 import com.google.firebase.firestore.FirebaseFirestore;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class Session {
     private FirebaseFirestore db;
     private User user;
     private static Session instance = null;
     private DocumentReference document;
-
+    private String emailEvent;
     /**
      * Singleton Session constructor
      * 
      * @param email, which is the document name in firestore
      */
     private Session(String email, Context context) {
+        emailEvent = email;
```

Comment:
```suggestion
    /**
     * Singleton Session constructor
     * 
     * @param email, which is the document name in firestore
     */
    private Session(String email, Context context) {
```
Doesn't look like these two are used, should be removed